apps/frontend-e2e/.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Is it smth included to the boilerplate out of the box? Or you plan to use Cypress in addition?)

Reply:
I didn't plan anything with cypress. It's just default nx repo 🙂